src/App.js

```diff
@@ -1,13 +1,28 @@
+import { useContext } from "react";
+
 import Header from "./components/Header";
 import Form from "./components/Form";
-// import NoTaskImage from "./components/NoTaskImage";
+import TaskList from "./tasks/TaskList";
+import NoTaskImage from "./components/NoTaskImage";
+import Toast from "./components/Toast";
+import DataContext from "./store/DataContext";
 
 function App() {
+    const data = useContext(DataContext); //using context
+
     return (
         <div>
             <Header />
             <Form />
-            {/* <NoTaskImage /> */}
+
+            {/* Shows Image and Text, only when there are no tasks */}
+            {data.tasks.length === 0 && <NoTaskImage />}
```

Comment:
While rendering use the Ternary operator, also equals-to zero check is not needed, since 0 is falsy in JS
```suggestion
            {data.tasks.length ? <NoTaskImage /> : null}
```